extension/index.js

```diff
@@ -102,8 +102,7 @@ async function openDropdown({currentTarget: indicator}) {
 			event.preventDefault();
 			const {form} = event.delegateTarget;
 			await postForm(form);
-
-			const notification = form.closest('.issue-notification');
+			const notification = form.closest('.js-notification');
```

Comment:
changed to `.js-notification` since it was common b/w PR notification and issue notification.